src/views/Home/ProjectItem/index.tsx

```diff
@@ -0,0 +1,288 @@
+            <div className={styles.name}>
+                {name}
+            </div>
+            <FormattedDate
+                className={styles.date}
+                // FIXME: Remove this fallback
+                date={date ?? Date.now()}
+                mode="hh:mmaaa, MMM dd, yyyy"
+                emptyComponent={emptyComponent}
+            />
+        </div>
+    );
+}
+
+const tickFormatter = (value: number | string) => {
+    const date = new Date(value);
+    return date.toDateString();
+};
+
+const minTickFormatter = (value: number | string) => {
+    const date = new Date(value);
+    return date.toDateString();
+};
+
+interface RecentProjectItemProps {
+    className?: string;
+    projectId: number;
+    title: string;
+    isPrivate: boolean;
+    startDate?: string;
+    endDate?: string;
+    description?: string;
+    projectOwnerName: string;
+    analysisFrameworkTitle?: string;
+    totalUsers: number;
+    totalSources: number;
+    totalSourcesTagged: number;
+    totalSourcesValidated: number;
+    projectActivity: CountTimeSeries[];
+    recentlyActive: UserActivityStat[];
+}
+
+const recentlyActiveKeySelector = (d: UserActivityStat) => d.id;
+
+function ProjectItem(props: RecentProjectItemProps) {
+    const {
+        className,
+        title,
+        startDate,
+        endDate,
+        isPrivate,
+        description,
+        projectId,
+        projectOwnerName,
+        analysisFrameworkTitle,
+        totalUsers,
+        totalSources,
+        totalSourcesTagged,
+        totalSourcesValidated,
+        recentlyActive,
+        projectActivity,
+    } = props;
+
+    const recentlyActiveRendererParams = useCallback((key, data) => ({
+        name: data.name,
+        date: data.date,
+    }), []);
+
+    const convertedProjectActivity = useMemo(() => (
+        projectActivity?.map(pa => ({
+            count: pa.count,
+            date: (new Date(pa.date)).getTime(),
+        })).sort((a, b) => compareDate(a.date, b.date))
+    ), [projectActivity]);
+
+    console.warn('here', props);
+
+    return (
+        <div className={_cs(className, styles.projectItem)}>
+            <header className={styles.header}>
+                <div className={styles.headerLeft}>
+                    <h2>
+                        {title}
+                    </h2>
+                    <div className={styles.dateContainer}>
+                        <FormattedDate
+                            title={_ts('home.recentProjects', 'startDateLabel')}
+                            date={startDate}
+                            mode="MMM yyyy"
+                            emptyComponent={emptyComponent}
+                        />
+                        {startDate && endDate && (
+                            <div className={styles.separator}>-</div>
+                        )}
+                        <FormattedDate
+                            title={_ts('home.recentProjects', 'endDateLabel')}
```

Comment:
same as above

Reply:
I forgot to attach the respective PR in react-store